utils/game-utils.js

```diff
@@ -0,0 +1,65 @@
+import { PLAYERS, DRAW_MATCH, WINNING_COMBINATIONS } from "../constants/game-constants.js";
+
+
+/**
+ * Function to determine the winner of a single match
+ * @param {*} player1Choice 
+ * @param {*} player2Choice 
+ * @returns winner of the match
+ */
+export const determineWinner = (player1Choice, player2Choice) => {
+    return player1Choice === player2Choice
+    ? DRAW_MATCH
+    : WINNING_COMBINATIONS[player1Choice] === player2Choice
+    ? PLAYERS.PLAYER_ONE
+    : PLAYERS.PLAYER_TWO;
+}
+
+/**
+ * Determine if the player already has a majority win
+ * @param {*} scores 
+ * @param {*} MAJORITY_WIN_COUNT 
+ * @returns winner of the game
+ */
+export const getMajorityChampion = (scores, MAJORITY_WIN_COUNT) => {
```

Comment:
could you explore if function definitions should have parameters in capital letters ?

Reply:
Usage of camel case in function parameters is the best practice